forkmanager/fork.go

```diff
@@ -0,0 +1,17 @@
+package forkmanager
+
+type ForkHandlerName string
+
+type ForkName string
```

Comment:
Why not simply using strings instead of type aliases? Though I agree that code is probably a bit more verbose this way.

Reply:
I just wanted to have possibility to easy switch to `int` maybe. But I am not even 90% sure. What other guys think?

Reply:
I have removed fork name alias because 99,99% fork name will always be string. But i have kept `HandlerDesc` alias because we may change it from string to int for performance gain